docs/index.md

```diff
@@ -1,17 +1,23 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: Tuition Address Book (TAB)
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/AY2122S1-CS2103T-F13-3/tp/branch/master/graph/badge.svg?token=ZHYAC5NIS7)](https://codecov.io/gh/AY2122S1-CS2103T-F13-3/tp)
 
-![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+> Are you a busy private 1-to-1 home tutor who finds it difficult to remember all your students' information?
```

Comment:
why is this a block quote 😂

Reply:
I wan to show engagement with target audience😂 @angkoonhwee

Reply:
imo it looks bad in block quote. Use bold or header for emphasis instead, our just do without?

Reply:
ok changed it